### DISCUSSION TITLE
115 网盘，电脑登录后，alist 里就被强制退出，无法挂载了

### DISCUSSION THREAD

**Discussion by xpairhub** [AlistGo/alist#5248] [Q&A]
每次都是 AList 里挂载了，PC 端被强制退出。反之，PC 端扫描登陆了，AList 端就无法挂载了。
是我一个人的问题吗？还是 115 网盘就是这样的呢？

**Comment by pongfcnkl** [comment]
抓115手机APP的cookie 放到alist上
然后电脑端网页端啥的用扫码登录就不会掉了

**Comment by free-150** [comment]
这里有115抓包的方法。抓包后时效很长
https://imnks.com/6088.html
